.github/workflows/go.yml

```diff
@@ -19,7 +19,7 @@ jobs:
     strategy:
       matrix:
         os: ["ubuntu-latest", "windows-latest"]
-        go: ["1.21.x", "1.22.x"]
+        go: ["1.22.x", "1.23.x"]
```

Comment:
```
    strategy:
      matrix:
        go: ["stable", "oldstable"]
```

and then

```
 go-version: ${{ matrix.go }}
```
+
```
if: matrix.latest
```
below

Reply:
Hmm... I don't have a big preference. Sometimes the latest version breaks some tests, and sometimes upgrading involves removing some old build-tagged tests (e.g., this PR). So in that vein it makes sense to make upgrading an explicit action/PR. On the other hand, this would reduce manual ops work.

Reply:
stable oldstable is also a valid option here. I'll defer to you on whether you want that or not.